py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -645,14 +644,11 @@ def aten_ops_softmax(
 
 
 @dynamo_tensorrt_converter(
-    torch.ops.aten.split.Tensor, capability_validator=dynamic_unsupported_with_args([1])
-)
-@dynamo_tensorrt_converter(
-    torch.ops.aten.split.sizes, capability_validator=dynamic_unsupported_with_args([1])
```

Comment:
Revisit the use of capability_validator here

Reply:
This one seems to be more of an edge case (it seems most converters can either support dynamic shapes, or not), perhaps the capability validator could be allowed to assess dynamic shape support in certain arguments for some challenging edge cases like this.